vaadin-grid-hierarchy-toggle.html

```diff
@@ -0,0 +1,142 @@
+<!--
+@license
+Copyright (c) 2017 Vaadin Ltd.
+This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+-->
+
+<link rel="import" href="../polymer/polymer-element.html">
+<link rel="import" href="../vaadin-themable-mixin/vaadin-themable-mixin.html">
+
+<dom-module id="vaadin-grid-hierarchy-toggle-default-theme">
+  <template>
+    <style>
+      [part="toggle"]::before {
+        display: inline-block;
+        width: 1em;
+        text-align: center;
+      }
+
+      :host(:not([expanded])) [part="toggle"]::before {
+        /* TODO: Enhance using icon font. */
+        content: "▸";
+      }
+
+      :host([expanded]) [part="toggle"]::before {
+        /* TODO: Enhance using icon font. */
+        content: "▾";
+      }
+
+      :host(:not([enabled])) [part="toggle"] {
+        visibility: hidden;
+      }
+    </style>
+  </template>
+</dom-module>
+
+<dom-module id="vaadin-grid-hierarchy-toggle">
+  <template>
+    <style>
+      :host([enabled]) {
+        cursor: pointer;
+      }
+
+      [part="spacer"] {
+        display: inline-block;
+        width: var(--level, '0em');
+      }
+    </style>
+
+    <span part="spacer"></span>
+    <span part="toggle"></span>
+    <span part="content">
+      <slot></slot>
+    </span>
+  </template>
+</dom-module>
+
+<script>
+  {
+    /**
+     * `<vaadin-grid-hierarchy-toggle>` is a helper element for the `<vaadin-grid>`
+     * that provides toggle and level display functionality for the item hierarchy.
+     *
+     * #### Example:
+     * ```html
+     * <vaadin-grid>
+     *   <vaadin-grid-hierarchy-toggle frozen></vaadin-grid-hierarchy-toggle>
```

Comment:
This example is outdated. The hierarchy toggle should be wrapped inside a regular column, right?

Reply:
Indeed, fixed